packages/components/textarea/textarea.component.ts

```diff
@@ -261,10 +281,11 @@ export class KbqTextarea
 
     /** Grow textarea height to avoid vertical scroll  */
     grow = () => {
-        if (!this.isBrowser || !this.canGrow) return;
+        if (!this.isBrowser || !this._canGrow) return;
```

Comment:
нужно ли учитывать maxRowLimitReached ?

```suggestion
        if (!this.isBrowser || !this.canGrow) return;
```

Reply:
не, тут специально `this._canGrow`.

Внутри `grow,` обновляется `this.rowsCount`, а он в свою очередь влияет на `this.maxRowLimitReached`, который в свою очередь влияет на `canGrow`